packages/wonder-blocks-cell/src/components/internal/cell-core.tsx

```diff
@@ -201,14 +177,28 @@ const CellCore = (props: CellCoreProps): React.ReactElement => {
                 hideDefaultFocusRing={true}
                 aria-label={ariaLabel ? ariaLabel : undefined}
                 target={target}
+                style={[
+                    styles.wrapper,
+                    styles.clickable,
+                    active && styles.active,
+                    disabled && styles.disabled,
+                ]}
+                aria-current={active ? "true" : undefined}
             >
-                {(eventState) => renderCell(eventState)}
+                {() => renderCell()}
             </Clickable>
         );
     }
 
     // No click event attached, so just render the cell as-is.
```

Comment:
> No click event attached, so just render the cell as-is.

Is this comment still needed?

Reply:
Nice catch! I'll delete it.

Reply:
Yeah, I think it's just to mention that we render the cell without any `Clickable` wrapper around. I'll update it to match this case.